JPS_Ontology/ontology/ontoems/OntoEMS.owl

```diff
@@ -1054,7 +1071,6 @@
 
 
     <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontoems/RelativeHumidity">
-        <owl:equivalentClass rdf:resource="https://www.auto.tuwien.ac.at/downloads/thinkhome/ontology/WeatherOntology.owl#Humidity"/>
```

Comment:
Where does this change come from? Doesn't seem to stem from the csv file...

Reply:
I think this is because I was using a newer version of the converter.